cypress/e2e/cart.cy.js

```diff
@@ -0,0 +1,66 @@
+describe('Error with no data',()=>{
+    it('Error wih no data',()=>{
+        cy.login('standard_user','secret_sauce')
+        cy.get('.shopping_cart_link').click()
+        cy.get('[data-test="checkout"]').click()
+        cy.get('[data-test="continue"]').click()
+        cy.get('.error-message-container').should('have.text','Error: First Name is required');
+    })
+})
+
+describe('Successful shopping flow',()=>{
+    it('Successful shopping flow',()=>{
+        cy.login('standard_user','secret_sauce')  
```

Comment:
when you add beforeEach, you don't need this line ( `cy.login('standard_user','secret_sauce')` ) here